Kinvey/Kinvey/MIC.swift

```diff
@@ -130,9 +153,13 @@ open class MIC {
     }
     
     @discardableResult
-    class func login<U: User>(refreshToken: String, client: Client = sharedClient, completionHandler: User.UserHandler<U>? = nil) -> Request {
+    class func login<U: User>(
+        refreshToken: String,
+        client: Client = sharedClient,
+        completionHandler: User.UserHandler<U>? = nil
+    ) -> Request {
         let requests = MultiRequest()
-        let request = client.networkRequestFactory.buildOAuthGrantRefreshToken(refreshToken: refreshToken)
+        let request = client.networkRequestFactory.buildOAuthGrantRefreshToken(refreshToken: refreshToken, clientId: nil)
```

Comment:
That's a very good question. Can we store the `client_id` along with the rest of the user information?